src/Traits/Hooks/UseDisplayDashboardTop.php

```diff
@@ -289,25 +170,31 @@ protected function shouldAttachRecommendedModules(string $type): bool
     }
 
     /**
-     * Returns the Help center link for the provided locale
+     * Add JS and CSS file
      *
-     * @param string $languageCode 2-letter locale code
+     * @see \PrestaShop\Module\Mbo\Traits\Hooks\UseAdminControllerSetMedia
      *
-     * @return string
+     * @return void
      */
-    protected function getHelpCenterLink(string $languageCode): string
+    protected function loadMediaForDashboardTop(): void
```

Comment:
This method is called in UseAdminControllerSetMedia. Does it belong to this trait ?

Reply:
Updated.
I've added a `boot{TraitName}` that is called in the module class in order to initialize methods and be sure everything is well declared before being called.
I think, with this method, each hook can add assets independently.

Reply:
Well done 👍🏾